nix/pkgs.nix

```
@@ -1,8 +1,11 @@
 # our packages overlay
 let
-  pkgs1903 = import (import ./sources.nix {})."nixpkgs-19.03" {};
+  sources = import ./sources.nix {};
+  pkgs1903 = import sources."nixpkgs-19.03" {};
 in pkgs: _: with pkgs; {
   jmPkgs = import ./jormungandr.nix { inherit (pkgs) commonLib; inherit pkgs; };
+  # TODO: do we need a system/config here somehow???
+  nodePkgs = import sources."cardano-node" {};
```

possibly
```suggestion
  nodePkgs = import sources."cardano-node" { inherit config system; };
```
but not how that would play with cross-compilation. Probably the cleanest would be to move this import directly to `nix/default.nix` and then import with all the arguments (system, crossSystem, config and sourcesOverride).